DQM/L1TMonitor/src/L1TRCT.cc

```diff
@@ -47,20 +48,6 @@ L1TRCT::L1TRCT(const ParameterSet & ps) :
 
   if (verbose_)
     std::cout << "L1TRCT: constructor...." << std::endl;
-
-  outputFile_ =
-      ps.getUntrackedParameter < std::string > ("outputFile", "");
-  if (!outputFile_.empty()) {
-    std::
-	cout << "L1T Monitoring histograms will be saved to " <<
-	outputFile_.c_str() << std::endl;
-  }
-
-  bool disable =
-      ps.getUntrackedParameter < bool > ("disableROOToutput", false);
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.